### PR TITLE
Add tests for patchanalysis.get_patch_info and for dataanalysis

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: libmozdata/config.py [hunk 1]
```diff
@@ -22,13 +22,17 @@ def __init__(self, path=''):
...
 
     def get(self, section, option, default=None):
         if not self.config.has_option(section, option):
             return default
 
         return self.config.get(section, option)
 
+    def __repr__(self):
+        return self.path
+
```
Comment (marco-c) [inline libmozdata/config.py]: Why these changes?

**Comment by marco-c** [comment]
Tests are failing because of `ImportError: No module named matplotlib`.
Can you add `matplotlib` to the requirements?

**Comment by sylvestre** [comment]
Merging #15 into master will increase coverage by 12.90%
nice work!
